src/build-disk-image.ts

```diff
@@ -67,6 +67,23 @@ export async function buildDiskImage(imageData: unknown, history: History) {
   const selectedType = selection.format;
   telemetryData.imageType = selectedType;
 
+  const selectionArch = await extensionApi.window.showQuickPick(
+    [
+      { label: 'ARM®', detail: 'ARM® 64 / aarch64 (arm64)', format: 'arm64' },
+      { label: 'AMD64', detail: 'AMD 64 / x86-64 (amd64)', format: 'amd64' },
```

Comment:
in Podman Desktop it has been replaced by 
- label `ARM64` with detail `ARM® aarch64 systems`
- label `AMD64` `Intel and AMD x86_64 systems`

I suppose we should have the same name there

Reply:
Agreed, updated.